### DISCUSSION TITLE
Power always off for switch?

### DISCUSSION THREAD

**Discussion by chrisharrington** [arendst/Tasmota#17771] [Q&A] [answered]
Hi there!

Is there any way to disable power control for a switch? I've got a room where I want the lighting to come from floor lamps and _not_ the overhead ceiling light. I've got a Martin Jerry MJ-S01 light switch hooked up that I'd like to use to control the floor lamps which are equipped with smart bulbs. So I hit the switch and Home Assistant takes care of toggling the smart bulbs in the floor lamps.

I've read through the Tasmota wiki and I wasn't able to find a command that would do the trick. I think the closest was `PowerOnState 4`, which turns the power on and disables power control. I want exactly the same thing as that but to have the power _off_ and then have power control disabled. I've also tried `SetOption114` and `SwitchMode 15` but no luck.

Are there are any commands or configuration that would do the trick for this? Or maybe some combination of rules? I'm new to all of this so the rules look a little daunting.

Thanks!

**Comment by barbudor** [comment]
Why not defining the GPIO as OUTPUT_LOW instead of Relay
That will leave the relay off and disale further control

Note that that will also disable the Led to follow the power state as there would be no more power state
If needed, you can still set another un-connected/un-used GPIO as Relay_1 (but still leave OUTPUT_LOW on GPIO12 ton insure real relay remains off)

**Comment by chrisharrington** [comment, marked as answer]
I ended up changing the template configuration so that the number after the relay is 2 from 1 and that solved my issue.
